index.js

```diff
@@ -12,6 +12,21 @@ function win(input, opts) {
 	});
 }
 
+function macOSKill(input, opts) {
+	const cmd = typeof input === 'string' ? 'pkill' : 'kill';
+	const args = [input];
+
+	if (opts.force) {
```

Comment:
All common prepping of args could be abstracted 🤔. Keep in mind there are three combinations of environments that might share a common `args` mutation based on the same option, that the third environment does not. I feel for now the complexity of an abstraction outweighs the code duplication, but I trust your judgment more on this.